shotwell: fails to compile

```
building path(s) ‘/nix/store/0v618xy13glbmfd4v3gcgknv1d9vf37j-shotwell-0.20.2’
unpacking sources
unpacking source archive /nix/store/694wz88m70zrry0l75zcypjmr5mlfkh6-shotwell-0.20.2.tar.xz
source root is shotwell-0.20.2
patching sources
configuring
patching script interpreter paths in .
./configure: interpreter directive changed from "/usr/bin/env bash" to "/nix/store/gb2dwphbdjk56h0lbbyma32rlid022ad-bash-4.3-p42/bin/bash "
./chkver: interpreter directive changed from " /bin/bash" to "/nix/store/gb2dwphbdjk56h0lbbyma32rlid022ad-bash-4.3-p42/bin/bash  "
configure flags: --prefix=/nix/store/0v618xy13glbmfd4v3gcgknv1d9vf37j-shotwell-0.20.2 --disable-gsettings-convert-install 
Detected libGPhoto 2.5.x - using 2.5-aware code path.
Configured.  Type 'make' to build, 'make install' to install.
building
make flags: SHELL=/nix/store/gb2dwphbdjk56h0lbbyma32rlid022ad-bash-4.3-p42/bin/bash    
Package gudev-1.0 was not found in the pkg-config search path.
Perhaps you should add the directory containing `gudev-1.0.pc'
to the PKG_CONFIG_PATH environment variable
No package 'gudev-1.0' found
Makefile:723: recipe for target 'valacheck' failed
make: *** [valacheck] Error 1
builder for ‘/nix/store/i2kvjr6kyhw7r4wdxb23gdqms34a0gfy-shotwell-0.20.2.drv’ failed with exit code 2
```

--- a/pkgs/applications/graphics/shotwell/default.nix
+++ b/pkgs/applications/graphics/shotwell/default.nix
@@ -1,5 +1,5 @@
 { fetchurl, stdenv, m4, glibc, gtk3, libexif, libgphoto2, libsoup, libxml2, vala, sqlite
-, webkitgtk24x, pkgconfig, gnome3, gst_all_1, which, udev, libraw, glib, json_glib
+, webkitgtk24x, pkgconfig, gnome3, gst_all_1, which, udev, libgudev, libraw, glib, json_glib
 , gettext, desktop_file_utils, lcms2, gdk_pixbuf, librsvg, makeWrapper
 , gnome_doc_utils, hicolor_icon_theme }:
 
@@ -37,7 +37,7 @@ stdenv.mkDerivation rec {
 
   buildInputs = [ m4 glibc gtk3 libexif libgphoto2 libsoup libxml2 vala sqlite webkitgtk24x
                   pkgconfig gst_all_1.gstreamer gst_all_1.gst-plugins-base gnome3.libgee
-                  which udev gnome3.gexiv2 hicolor_icon_theme
+                  which udev libgudev gnome3.gexiv2 hicolor_icon_theme
                   libraw json_glib gettext desktop_file_utils glib lcms2 gdk_pixbuf librsvg
                   makeWrapper gnome_doc_utils gnome3.rest
                   gnome3.defaultIconTheme ];

